Sources/Swift/Protocol/Codable/SentryLogAttributeCodable.swift

```diff
@@ -0,0 +1,42 @@
+extension SentryLog.Attribute: Codable {
+    
+    private enum CodingKeys: String, CodingKey {
+        case value
+        case type
+    }
+    
+    public func encode(to encoder: any Encoder) throws {
```

Comment:
`h`: Please don't make all the `encode` and `init` methods public. We only did this for the other classes because they're adding decodable implementations for public ObjC classes and they therefore must be public. The `SentryLog.Attribute` isn't public, so no need to make this public please.